chronos2/dashboard_backend/src/core/configs/database.py

```diff
@@ -8,14 +8,14 @@
 
 load_dotenv()
 
-DB_USER = os.getenv("DB_USER")
-DB_PASSWORD = os.getenv("DB_PASSWORD")
+DB_USER = os.getenv("POSTGRES_USER")
+DB_PASSWORD = os.getenv("POSTGRES_PASSWORD")
 DB_HOST = os.getenv("DB_HOST")
 DB_PORT = os.getenv("DB_PORT")
-DB_NAME = os.getenv("DB_NAME")
+DB_NAME = os.getenv("POSTGRES_DB")
 
 # Replace with your own PostgreSQL instance
-DATABASE_URL = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+DATABASE_URL = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/postgres"
```

Comment:
I've changed DB_NAME to postgres. Changing the name using the dockerized postgres is a bit of a pain so let's just keep it hard-coded.
Also DB_NAME wasn't in the .env.sample file.